spec/features/ncr/work_orders/requester_edit_spec.rb

```diff
@@ -7,7 +7,7 @@
     end
   end
 
-  let(:work_order) { create(:ncr_work_order, description: 'test') }
+  let(:work_order) { create(:ncr_work_order, org_code: Ncr::Organization.all[0].to_s,  description: 'test') }
```

Comment:
Ok so I just tried that route and it was kind of a pain because this is a non-AR class. But if we are going to make this an AR class (via https://github.com/18F/C2/pull/814) then it will become much easier. So I am gonna work on that and return to this PR afterward. Thanks for the suggestion!